practice-areas/design-and-research/README.md

```diff
@@ -1,41 +1,102 @@
 
-We use metrics to gauge impact and guide decisions
+### We approach our work with authentic empathy
+
+-   We acknowledge each other's whole self
+-   We can show care for others when we show care for ourselves
+-   We acknowledge that we all work in different ways and we respect diversity in thought, approach, and execution
+
+### Good enough should not be our default
+
+-   We strive to fix what is broken
+-   We advocate for, and educate others, on design best practices
+-   We seek to find solutions for all of humanity, not just a subset of users
+-   We have a growth mindset that is open to constructive feedback
+
+### We achieve through imagination and iteration
+
+-   We push ourselves and each other to go beyond what we think we're capable of
+-   We know that success requires collaboration and teamwork
+-   We use metrics to gauge impact and guide decisions
+
+## All creatives calls
+
+We meet regularly on Zoom to discuss topics of common interest. Mostly we tackle design ops topics during these meetings: what are our design principles, what are our latest engagement scores, how to best promote yourself and your work, etc. However, we do occasionally use these meetings for team-building activities and design critiques. These meetings always include an agenda when they're scheduled and are recorded so you can best decide how to prioritize your time. This also makes sure you can still get the information from the meeting if you have to miss it. While the recordings are shared after each meeting, we encourage you to attend the meeting if possible.
+We add the agenda recap, slide decks, and recordings to our [ACC Notes](https://civicactions.atlassian.net/wiki/spaces/DES/pages/94044176/Meetings+Connection) after each meeting.
+
+## Internal initiatives
+
+Our design team spends most of our time on billable projects, but we recognize that, as designers, we often look for things we can create or enhance on our own. As designers, we often get asked to create deliverables for our marketing, project, and product teams here at CivicActions. We work on these internal projects when we have the time, remembering to prioritize our billable work first. Because our design work tends to ebb and flow, you may not finish an internal project you start on, and that's ok! We ask that you update the work with your progress and next steps so someone else can pick it up as they have time.
+
+## Professional development
+
+We've included some possible professional development resources below. You have $1200 per year (after your 90-day intro period) to spend on [professional development](../../professional-development/README.md). Professional development conferences, courses, and webinars that align with your current or growth-level path can contribute to your yearly utilization targets, but you don't have to use your professional development for our practice area only.
```

Comment:
@kellysmith1008 ah this one is getting caught in the problem because the file is `professional-development.md` instead of a README path.

Reply:
Sorry for the trouble here @kellysmith1008. I believe the full link to that file is:

`[professional development](../../employee-benefits/professional-development.md)`

Reply:
Ah crapballs. I'll get there eventually! Thank you for your patience and help!

Reply:
No worries. I believe you got it.